risor.go

```diff
@@ -33,6 +34,75 @@ func WithGlobal(name string, value any) Option {
 	}
 }
 
+// WithoutGlobal opts out of a given global builtin or module. If the name can't
+// be resolved, this is a no-op. This does operate on nested modules.
+func WithoutGlobal(name string) Option {
+	return func(cfg *Config) {
+		parts := strings.SplitN(name, ".", 2)
+		if len(parts) == 1 {
+			delete(cfg.Globals, name)
+			delete(cfg.DefaultGlobals, name)
+			return
+		}
+		// Resolve the module (which could be nested) and then remove the
+		// named attribute from it. This could be in either or both of
+		// the Globals and DefaultGlobals structures.
+		moduleName, attr := parts[0], parts[1]
+		if obj, ok := cfg.Globals[moduleName]; ok {
+			if m, ok := obj.(*object.Module); ok {
+				removeModuleAttr(m, attr)
+			}
+		}
+		if obj, ok := cfg.DefaultGlobals[moduleName]; ok {
+			if m, ok := obj.(*object.Module); ok {
+				removeModuleAttr(m, attr)
+			}
+		}
+	}
+}
+
+// WithoutGlobals removes multiple global builtins or modules.
+func WithoutGlobals(names ...string) Option {
+	return func(cfg *Config) {
+		for _, name := range names {
+			WithoutGlobal(name)(cfg)
+		}
+	}
+}
+
+// WithGlobalOverride replaces the a global or module builtin with the given value
+func WithGlobalOverride(name string, value any) Option {
+	return func(cfg *Config) {
+		parts := strings.Split(name, ".")
+		if len(parts) == 1 {
+			cfg.Globals[name] = value
+			delete(cfg.DefaultGlobals, name)
+			return
+		}
+		valueObj := object.FromGoType(value)
+		if valueObj == nil || valueObj.Type() == object.ERROR {
+			panic(fmt.Sprintf("invalid value for global override: %v", value))
+		}
+		moduleName := parts[0]
+		nestedModulePath := parts[1 : len(parts)-1]
+		attrName := parts[len(parts)-1]
+		if obj, ok := cfg.Globals[moduleName]; ok {
+			if m, ok := obj.(*object.Module); ok {
+				if targetMod, ok := resolveModule(m, nestedModulePath); ok {
+					targetMod.SetAttr(attrName, valueObj)
```

Comment:
can't it be dangerous to allow doing this through `SetAttr`? that was why in my implementation I had a dedicated `Override` method that wasn't accessible from `risor` code directly.

Reply:
You're thinking of the situation when Risor is being used in a sandboxing-like capability, right? 

I'm doing a quick review of things from this point of view. Will add back `Override` depending on how this all looks.

Reply:
> You're thinking of the situation when Risor is being used in a sandboxing-like capability, right?

yes

Reply:
I've used OPA extensively, so this sounds familiar :-)